src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.builds

```diff
@@ -4,19 +4,17 @@
   <ItemGroup>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Unix</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
-      <TestTFMs>net463</TestTFMs>
```

Comment:
Actually looks like we might not even need the Unix build for netstandard1.3 as it doesn't look like it is being used in the test project. We at least need to add the net463 configuration with Windows_NT as they will not work or run on Unix.